packages/dev-tools/client/package.json

```
@@ -0,0 +1,51 @@
+{
+  "main": "node_modules/expo/AppEntry.js",
+  "scripts": {
+    "start": "expo start --web",
```

```suggestion
    "start": "expo start:web",
```
prevent metro from starting